### PR TITLE
Updates rubocop config

### PR DESCRIPTION
**Pull request by ascoppa**
Minor updates to match our preferences (according to what has been discussed)

### CODE REVIEW

File: config/rubocop-2.4.yml [hunk 1]
```diff
@@ -103,6 +103,10 @@ Layout/EmptyLines:
   Description: Don't use several empty lines in a row.
   Enabled: true
 
+Layout/EmptyLineAfterGuardClause:
```
Comment (jackchoi) [inline config/rubocop-2.4.yml]: This should be above `EmptyLineBetweenDefs` since it's alphabetical

File: config/rubocop-2.4.yml [hunk 2]
```diff
@@ -567,6 +571,9 @@ Metrics/PerceivedComplexity:
   Max: 30
 
 ##################### Naming ##################################
+Naming/AccessorMethodName:
```
Comment (jackchoi) [inline config/rubocop-2.4.yml:575]: Normally an empty line after the comment banner

File: config/rubocop-2.4.yml [hunk 3]
```diff
@@ -608,6 +615,10 @@ Naming/MethodName:
   - snake_case
   - camelCase
 
+Naming/MemoizedInstanceVariableName:
```
Comment (jackchoi) [inline config/rubocop-2.4.yml]: Before `Naming/MethodName` alphabetically

File: config/rubocop-2.4.yml [hunk 4]
```diff
@@ -1342,5 +1361,5 @@ Style/WordArray:
   Description: Use %w or %W for arrays of words.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#percent-w
   Enabled: true
-  MinSize: 0
+  MinSize: 3
```
Comment (jackchoi) [inline config/rubocop-2.4.yml]: Interesting we already had 0, maybe we should keep it then? I was under the impression this was a new thing

File: config/rubocop-2.4.yml (hunk 1 above)
Comment (ascoppa) [reply config/rubocop-2.4.yml]: Definitely, I'll make sure to double check the other cops as well.

File: config/rubocop-2.4.yml (hunk 4 above)
Comment (ascoppa) [reply config/rubocop-2.4.yml]: My bad, this is also present on the rubocop file targeting ruby 2.1 https://github.com/bespokepost/code-climate-config/blob/master/config/rubocop.yml#L1346 

I will undo.

**Comment by ascoppa** [comment]
@jackchoi this is ready for another shot https://github.com/bespokepost/code-climate-config/pull/6/commits/effcaefeb0edecba419c501b4ddf29e806de6d43

**Review by jackchoi — APPROVED** [review]

**Pull request merged**